rt/wsdl/pom.xml

```diff
@@ -119,4 +117,19 @@
             </plugin>
         </plugins>
     </build>
+    
+    <profiles>
+        <profile>
+            <id>xerces</id>
+            <activation>
+                <jdk>[9,)</jdk>
+            </activation>
+            <dependencies>
+                <dependency>
```

Comment:
@ffang Another option I've been considering is opt-in model, remove profile and have `xercesImpl` as optional dependency:

```
               <dependency>
                    <groupId>xerces</groupId>
                    <artifactId>xercesImpl</artifactId>
                    <optional>true</optional>
                </dependency>
```